unfoldingWord/translationCore
# Project Validation Stepper Main Container (Import Check)

#### This pull request addresses:
This PR contains the basic containers, components, actions and reducers for handling project validation elements.


#### How to test this pull request:
Try and open a project and make sure the stepper shows, and you should be able to go through each step until the end.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2386)
